tools/testsys/Test.toml.example

```diff
@@ -0,0 +1,78 @@
+
+# The uri for the EKS resource agent that should be used.
+# An individual agent's provided URI will be used even if `testsys-images-registry` is present.
+eks-resource-agent-image = "public.ecr.aws/bottlerocket-test-system/eks_resource_agent:v0.0.2"
+
+# The test configuration is broken up into different levels depending on the variant.
+# Each configuration's value will be selected in the following order:
+# Arguments from the command line
+# Environment variables from the command line
+# '{variant}' configuration in `Test.toml`
+# '{family}-{flavor}' configuration in `Test.toml`
+# '{family}' configuration in `Test.toml`
+# '{platform}-{flavor}' configuration in `Test.toml`
+# '{platform}' configuration in `Test.toml`
+# TestSys default values
+
+# Configuration for all AWS variants (platform level configuration).
+[aws]
+agent-role = "<aws-testing-role>"
+
+# Configuration for all nvidia AWS variants on x86_64 (platform-flavor level configuration)
+[aws-nvidia.x86_64]
+instance-type = "p3.2xlarge"
+
+# Configuration for all nvidia AWS variants on aarch64 (platform-flavor level configuration)
+[aws-nvidia.aarch64]
+instance-type = "g5g.2xlarge"
+
+# Configuration for all `aws-k8s` variants testing (family level configuration).
+[aws-k8s]
+# A single role can be assumed by agents to test all `aws-k8s` variants in a separate
+# testing account.
+agent-role = "arn:aws:iam::<account_id>:role/<role_name>"
+
+# To test sonobuoy certified conformance in other AWS partitions, sonobuoy e2e
+# images need to be available in a registry in the targeted region (cn-north-1, us-gov-west-1).
+e2e-repo-registry = "<account>.dkr.ecr.cn-north-1.amazonaws.com.cn"
+
+# If testing using a kind cluster, AWS credentials need to be passed as a K8s secret.
+secrets = {"awsCreds" = "myAwsCredentials"}
+
+# Configuration for all nvidia AWS variants on x86_64 (family-flavor level configuration)
+[aws-ecs-nvidia.x86_64]
+instance-type = "p3.2xlarge"
+
+# Configuration for all nvidia AWS variants on aarch64 (family-flavor level configuration)
+[aws-ecs-nvidia.aarch64]
+instance-type = "g5g.2xlarge"
+
+# Configuration for only the `aws-k8s-1.21` variant (variant level configuration).
+["aws-k8s-1.21"]
+x86_64 = {}
+aarch64 = {kube-conformance-image = "<kube-conformance-image-uri>"}
+
+# Configurable fields:
+# cluster-names: 
+#     For `aws-k8s` variants testing will be completed over all clusters listed.
+#     For `aws-ecs` variants, only the first cluster name will be used.
+#     Cluster naming supports handlebars for `arch` and `variant` (`{{arch}}-{{variant}}`).
+# secrets:
+#     A map containing all secrets needed for resource creation and testing.
+# agent-role:
+#     The role that should be assumed by each test and resource agent.
+# kube-conformance-image: (K8s only)
+#     Specify a custom Kubernetes conformance image for E2E sonobuoy testing.
+# e2e-repo-registry: (K8s only)
+#     Configure the registry used for Sonobuoy E2E testing images.
```

Comment:
Should we make these fields more general, e.g. `conformance-image` and `conformance-registry`?